js/models/Profile.js

```diff
@@ -84,17 +54,23 @@ export default class extends BaseModel {
     colorFields.forEach((colorField) => {
       const clr = attrs[colorField];
 
-      if (typeof clr !== 'undefined' && is.not.hexColor(clr)) {
-        addError(colorField, 'Please provide a valid hex color.');
+      if (is.not.hexColor(clr)) {
+        addError(colorField, app.polyglot.t('profileModelErrors.provideValidHexColor'));
+      } else if (clr.charAt(0) !== '#') {
+        addError(colorField, 'The color should start with a leading hash.');
       }
     });
 
     if (attrs.email && is.not.email(attrs.email)) {
-      addError('email', 'Please provide a valid email.');
+      addError('email', app.polyglot.t('profileModelErrors.provideValidEmail'));
     }
 
     if (attrs.website && is.not.url(attrs.website)) {
-      addError('website', 'Please provide a valid url.');
+      addError('website', app.polyglot.t('profileModelErrors.provideValidURL'));
+    }
+
+    if (attrs.handle && attrs.handle.charAt(0) === '@') {
+      addError('handle', 'The handle should not start with a leading hash.');
```

Comment:
There's a few that I purposely left untranslated if they're ones that will never reasonably make it into the UI. I don't want to clutter up the translations file in that case.

For example, in the settings form, I'm having the view strip out any leading @, so the user will never see this error.

Same goes for the leading hash on the color field.

Anyhow, if you still think it's worth it to translate it, even though it's very likely not make it into the UI, I could do so..?

(I will update the handle error to reference '@' instead of hash.)